templates/web/panels/panel.tt.html

```diff
@@ -175,6 +175,21 @@ <h4 class="panel_title">
                 <hr class="floatclear">
                 [% END %]
 
+            [% ELSIF element_type == "action" %]
+                <div>
+                [% action_element = element_ref.action_element %]
+
+                [% action_element.html %]
+
+                [% FOREACH action IN action_element.actions %]
+                    <a class="button small"
```

Comment:
maybe also add action name as a class ?
like action-add_categories

This can help debug, trigger js or so, in the future.

```suggestion
                    <a class="button small action-[% action %]"
```

Reply:
Sounds good.